src/LetsEncrypt/Internal/LoggerExtensions.cs

```diff
@@ -8,14 +8,14 @@ namespace McMaster.AspNetCore.LetsEncrypt.Internal
 {
     internal static class LoggerExtensions
     {
-        public static void LogResponse<T>(this ILogger logger, string actionName, AcmeResult<T> response)
+        public static void LogAcmeAction(this ILogger logger, string actionName, object result)
```

Comment:
`result` isn't used...which IMO makes the logging a little less useful. Is there no equivalent to `AcmeResult<T>` in the v2 API? I seek to avoid `object` parameters when possible.

Reply:
Not sure what to do with this one. If there is an error it throws an exception, if it is success we get a model but they have no common ancestor. Also some of the models contain secrets so it is not a good idea to log them. I'm keeping it like this and you can do something with it. Maybe delete it altogether?

Reply:
Deleting altogether is probably best. I'll take care of that later....that's just a code cleanup, not essential to finishing this features.